docs/requirements-specification.md

```diff
@@ -21,4 +21,42 @@ The use cases fulfill the goals in the following way:
 | UC5 | G4, G8 |
 | UC6 | |
 
-The goals G10-G12 are currently not addressed by any use case.
\ No newline at end of file
+The goals G10-G12 are currently not addressed by any use case.
+
+### UC3: Get capability overview
+| UC3 | Get capability overview
+|---|---|
+| Actors | (Senior) Researcher |
+| Pre-conditions | The researcher is authenticated |
+| End conditions (success) | An overview of the capabilities of a research group is generated. |
+| End conditions (success) | An overview of the capabilities of a research group is generated. |
+| Steps | 1. If (Senior) Researcher presses "Research group overview" the system opens a new window with a table containing the following information about all the researchers belonging to the research group: researcher name, capability groups in which this researcher has capabilities, capabilities this researcher has, link to profile page. |
+| | 2. If (Senior) Researcher clicks on the page "Generate visualization" a simple visualization (piechart) of a proportion of table entries in columns capability groups, capabilities is shown. |
+| Extensions | 1.a. The (Senior) Researcher can filter table entries in each column. |
+| | 1.b. The (Senior) Researcher can sort table entries in each column. |
+
+
+
+| UC3 | Get capability overview
+|---|---|
+| Actors | Researcher |
+| Pre-conditions | The researcher is authenticated |
+| End conditions (success) | A link to a page with customized portfolio visualization is generated. |
+| End conditions (success) | A link to a page with customized portfolio visualization is not generated. |
+| Steps | 1. If the researcher clicks on "Visualize portfolio", a new page opens that contains profile information. |
+| | 2. The Researcher selects information that he would like to visualize (capability group, capability, evidence). |
+| | 3. The system generates a preview of customized portfolio visualization.
+| | 4. If the researcher clicks on "Save profile", a dialog window appears containing the URL of the page with personal capability portfolio visualization. |
+
+
+| UC3 | Get capability overview
```

Comment:
This UC overlaps with UC5 and would introduce redundancy. There is no need for this UC in the scope of UC3.